City-of-Helsinki/drupal-helfi-strategia
# UHF-2180: Updated base configuration

How to test:

- Checkout this branch.
- Run `make drush-cim && make drush-cr`.
- Check that the site includes the latest features and correct name.
